test/e2e/e2e_suite_test.go

```diff
@@ -101,6 +101,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	// Before all ParallelNodes.
 
 	Expect(configPath).To(BeAnExistingFile(), "Invalid test suite argument. e2e.config should be an existing file.")
+    Expect(os.RemoveAll(artifactFolder)).To(Succeed())
```

Comment:
Indentation looks strange here. This line is not at the same level as the others.